### PR TITLE
Use fix name for mcvForInstallPlan instead of safeName

### PR DESCRIPTION
**Pull request by sudomakeinstall2**
Signed-off-by: Saeid Askari <saskari@redhat.com>
closes: https://issues.redhat.com/browse/CNF-7147

### CODE REVIEW

File: controllers/utils/multicloud_util.go [hunk 1]
```diff
@@ -224,20 +220,20 @@ func EnsureManagedClusterView(
...
 		// If the specific managedClusterAction was not found, create it.
 		if errors.IsNotFound(err) {
 			multiCloudLog.Info("[EnsureManagedClusterActionForInstallPlan] MCA doesn't exist, create it",
-				"name", safeName, "namespace", namespace)
+				"name", installPlan.Name, "namespace", namespace)
 			actionMeta := metav1.ObjectMeta{
-				Name:      safeName,
+				Name:      installPlan.Name,
 				Namespace: namespace,
 				Annotations: map[string]string{
```
Comment (jc-rh) [inline controllers/utils/multicloud_util.go:235]: This annotation is no longer needed.

Comment (sudomakeinstall2) [reply controllers/utils/multicloud_util.go]: updated

**Review by jc-rh — APPROVED** [review]

**Pull request merged**